### ISSUE TITLE
[quilt] Foxes do not run at all 1.0.1

### ISSUE THREAD

**Issue by V-lpen** [TalonFloof/Vulpine#2]
Entirely untamed foxes will not run from me

using quilt 1.20.1

**Comment by Daelyx** [comment]
Yeah... before they were running even with sneak approach, now that it's fixed they're just not running at all ^^'

**Comment by Daelyx** [comment]
that's true for Fabric too, btw

**Comment by V-lpen** [comment]
Seems like they also dont want to sleep?
Maybe im just unlucky but they arent sleeping

**Comment by V-lpen** [comment]
Hi there I love this mod a lot, but the bug completely takes away from fox behaviors. any hope for a fix?

**Comment by TalonFloof** [comment]
Not sure tbh, I've never been able to get to the core reason for why this is happening.

**Comment by TalonFloof** [comment]
Fixed in latest commit, will release update tomorrow
